src/Features/CSharp/Portable/CodeFixes/AddImport/CSharpAddImportCodeFixProvider.cs

```diff
@@ -390,76 +385,100 @@ protected override string GetDescription(INamespaceOrTypeSymbol namespaceSymbol,
 
         protected override async Task<Document> AddImportAsync(
             SyntaxNode contextNode,
-            INamespaceOrTypeSymbol namespaceSymbol,
+            INamespaceOrTypeSymbol namespaceOrTypeSymbol,
```

Comment:
Note: this file didn't change much.  Most of it is an extract method + indent.  Just github displays it badly.  All that's really happening is that if we're adding a using for a spelling mistake, we also fix up your spelling mistake at hte same time.